packages/myst-cli/src/transforms/inlineExpressions.ts

```diff
@@ -16,8 +16,8 @@ export interface IBaseExpressionResult {
 
 export interface IExpressionOutput extends IBaseExpressionResult {
   status: 'ok';
-  data: Record<string, string>;
-  metadata: Record<string, string>;
+  data: Record<string, any>;
+  metadata: Record<string, any>;
```

Comment:
MIME bundles should have untyped values, as AFAICT there's no requirement for the content to be a string.